back-end/server/api/users.js

```diff
@@ -36,23 +36,23 @@ router.get('/:userId/activities', async (req, res, next) => {
   }
 })
 
-router.get('/:userId/friends', async (req, res, next) => {
+router.get('/:userId/followees', async (req, res, next) => {
   try {
-    const {Friends} = await User.findOne({
+    const {Followees} = await User.findOne({
       where: {
         id: req.params.userId
       },
-      include: ['Friends']
+      include: ['Followees']
     })
-    const friends = await User.friendsInAlphabeticalOrder(Friends);
-    res.json(friends)
+    const followees = await User.followeesInAlphabeticalOrder(Followees)
+    res.json(followees)
   } catch (err) {
     next(err)
   }
 })
 
-router.get('/:userId/friends/:friendId', async (req,res,next) => {
-  try{
+router.get('/:userId/friends/:friendId', async (req, res, next) => {
```

Comment:
Did you miss changing some of the routes in this file? some of the routes still say friends